AnthroFace/rib-fracture
# Henry

## Purpose
Importer now gives warnings when a patient can't be imported stating why! They aren't the prettiest but they should be descriptive enough.  

Importer also now aborts import if the headers are not formatted correctly. Exporter template will need to be updated to include the proper header format.

#### Steps for testing (if needed)
- Make sure all the errors given are correct and make sense
- Test that improperly formatted CSV files are not parsed
- Test that properly formatted CSV files are parsed

## Learning
- Proper headers are slightly different than the ones on CSV files that are given.  A small CSV file with proper headers will be put in the resources channel on Slack for testing purposes
